tests/qos/qos_sai_base.py

```diff
@@ -1058,7 +1058,7 @@ def dutQosConfig(
             "portSpeedCableLength": portSpeedCableLength,
         }
 
-    @pytest.fixture(scope='class')
+    @pytest.fixture(scope='function')
```

Comment:
There was a fix put in long time ago for this issue. Has anything changed recently?
https://github.com/sonic-net/sonic-mgmt/pull/5452

Reply:
Thanks Neetha,

need a little bit changes for PR#5452, because dut choosing need to match mux  cable toggle behavior in dualtor.
and clear arp and fdb table before ARPpopulate()

will update changes in this pr.